Description: Instead of saying X-Leave, say Leave
 There's no Wayland-Leave, is there?
Author: Simon Quigley <tsimonq2@ubuntu.com>
Origin: vendor
Forwarded: no
Last-Update: 2023-11-13
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/menu/lxqt-applications-compact.menu
+++ b/menu/lxqt-applications-compact.menu
@@ -209,7 +209,7 @@
 
 	<!-- Leave -->
 	<Menu>
-		<Name>X-Leave</Name>
+		<Name>Leave</Name>
 		<Directory>lxqt-leave.directory</Directory>
 		<Include>
 			<And>
@@ -240,7 +240,7 @@
 		<Menuname>LXQtSettings</Menuname>
 		<Menuname>DesktopSettings</Menuname>
 		<Separator/>
-		<Menuname show_empty="false" inline="true">X-Leave</Menuname>
+		<Menuname show_empty="false" inline="true">Leave</Menuname>
 		<Menuname show_empty="false" inline="true">Screensaver</Menuname>
 	</Layout>
 
--- a/menu/lxqt-applications-simple.menu
+++ b/menu/lxqt-applications-simple.menu
@@ -193,7 +193,7 @@
 
 	<!-- Leave -->
 	<Menu>
-		<Name>X-Leave</Name>
+		<Name>Leave</Name>
 		<Directory>lxqt-leave.directory</Directory>
 		<Include>
 			<And>
@@ -217,7 +217,7 @@
 		<Separator/>
 		<Menuname show_empty="false" inline="true">Configuration Center</Menuname>
 		<Separator/>
-		<Menuname show_empty="false">X-Leave</Menuname>
+		<Menuname show_empty="false">Leave</Menuname>
 		<Menuname show_empty="false" inline="true">Screensaver</Menuname>
 	</Layout>
 
--- a/menu/lxqt-applications.menu
+++ b/menu/lxqt-applications.menu
@@ -205,7 +205,7 @@
 
 	<!-- Leave -->
 	<Menu>
-		<Name>X-Leave</Name>
+		<Name>Leave</Name>
 		<Directory>lxqt-leave.directory</Directory>
 		<Include>
 			<And>
@@ -229,7 +229,7 @@
 		<Menuname>DesktopSettings</Menuname>
 		<Menuname show_empty="false" inline="true">LXQt-About</Menuname>
 		<Separator/>
-		<Menuname show_empty="false">X-Leave</Menuname>
+		<Menuname show_empty="false">Leave</Menuname>
 		<Menuname show_empty="false" inline="true">Screensaver</Menuname>
 	</Layout>
 
